src/__test__/utils.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { getAgeRangesForLifeStage, getPercentDaily, spaceToDashes } from "../helpers/utils";
+// import { render } from "@testing-library/react";
+import { RdiAge } from "../models";
+
+describe("Utils Testing", () => {
```

Comment:
Move test file to where the source file is.

Reply:
done